### PR TITLE
[IOTDB-240] fix unknown time series in where clause

### PR DESCRIPTION
**Pull request by SilverNarcissus**
fix unknown time series in where clause and add test

### CODE REVIEW

**Comment by jt2594838** [comment]
Pay attention to the format of the title.

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java [hunk 1]
```diff
@@ -334,4 +334,34 @@ public void crossSeriesReadUpdateTest() throws ClassNotFoundException, SQLExcept
...
+    Class.forName(Config.JDBC_DRIVER_NAME);
+
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      statement.execute("select s10 from root.vehicle.d0");
+      fail("not throw exception when select unknown time series");
+    } catch (SQLException e) {
+      assertEquals("Execute statement error: Path: \"root.vehicle.d0.s10\" not corresponding any known time series", e.getMessage());
+      e.printStackTrace();
```
Comment (jt2594838) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java]: Remove the stack print.

File: server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java [hunk 2]
```diff
@@ -334,4 +334,32 @@ public void crossSeriesReadUpdateTest() throws ClassNotFoundException, SQLExcept
...
+  }
+
+  @Test
+  public void selectWhereUnknownTimeSeries() throws ClassNotFoundException {
+    Class.forName(Config.JDBC_DRIVER_NAME);
+
+    try (Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+        Statement statement = connection.createStatement()) {
+      statement.execute("select s1 from root.vehicle.d0 where s0 < 111 and s10 < 111");
```
Comment (LeiRui) [inline server/src/test/java/org/apache/iotdb/db/integration/IoTDBMultiSeriesIT.java]: Better add a test for the case where filter paths start with ROOT. Correspond to the error exception you add in the class `MTree`.

File: server/src/main/java/org/apache/iotdb/db/qp/strategy/optimizer/ConcatPathOptimizer.java [hunk 3]
```diff
@@ -343,7 +346,7 @@ private void removeStarsInPath(List<Path> paths, List<String> afterConcatAggrega
       try {
         List<String> actualPaths = executor.getAllPaths(paths.get(i).getFullPath());
         if(actualPaths.isEmpty()){
-          throw new LogicalOptimizeException("Path: \"" + paths.get(i) + "\" not corresponding any known time series");
+          throw new LogicalOptimizeException("Path: \"" + paths.get(i) + "\" doesn't corresponding to any known time series");
```
Comment (jt2594838) [inline server/src/main/java/org/apache/iotdb/db/qp/strategy/optimizer/ConcatPathOptimizer.java]: not "ing"

**Pull request merged**